Index.html

```diff
@@ -99,14 +103,18 @@ <h2>Skills</h2>
                     <img src="./images/Vector4.png" alt="vector">
                 </div>
                 <div class="socialContent">
```

Comment:
- [ ] Kindly add appropriate padding before this div, check Figma design

Reply:
@prolajumokeoni: Kindly, please be more specific about what div you are referring to, this is the only problem I have pending to correct.

Reply:
Hello @NeckerFree